examples/demo-app/src/app.js

```diff
@@ -70,6 +78,7 @@ class App extends Component {
   };
 
   componentWillMount() {
+    this.props.dispatch(initApp());
```

Comment:
init only set `loaded:true` it is a placeholder action, and is not used anywhere.

Reply:
done